.github/workflows/terraform-tools.yml

```diff
@@ -0,0 +1,42 @@
+name: terraform-tools
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  tfsec:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - uses: actions/checkout@v3.4.0
+      - run: rm -r modules/ # remove modules from being security checked
+      - run: rm -r terraform/ # remove legacy terraform dir from being security checked
+      - uses: aquasecurity/tfsec-sarif-action@v0.1.4
+        with:
+          tfsec_args: --force-all-dirs --soft-fail -m CRITICAL
+          sarif_file: tfsec.sarif
+      - uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: tfsec.sarif
+  tflint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.4.0
+      - uses: actions/cache@v3.3.1
+        with:
+          path: ~/.tflint.d/plugins
+          key: tflint-${{ hashFiles('.tflint.hcl') }}
+      - uses: terraform-linters/setup-tflint@v3.0.0
+        with:
+          tflint_version: latest
+      - run: tflint --version
+      - run: tflint -c $(realpath .tflint.hcl) --init
+        env:
+          GITHUB_TOKEN: ${{ github.token }} # https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md#avoiding-rate-limiting
```

Comment:
From https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

> A token to authenticate on behalf of the GitHub App installed on your repository. This is functionally equivalent to the GITHUB_TOKEN secret. For more information, see "[Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication)."
Note: This context property is set by the Actions runner, and is only available within the execution steps of a job. Otherwise, the value of this property will be null.